queries.sql

```diff
@@ -5,4 +5,52 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
```

Comment:
- [ ] Please you missing the queries for these two requirement :point_down: so make sure the queries are present in this file :pray: 
    
    - Inside a transaction update the `animals` table by setting the `species` column to `unspecified`. Verify that change was made. Then roll back the change and verify that the `species` columns went back to the state before the transaction.
    - Inside a transaction:
      - Update the `animals` table by setting the `species` column to `digimon` for all animals that have a name ending in `mon`.
      - Update the `animals` table by setting the `species` column to `pokemon` for all animals that don't have `species` already set.
      - Verify that changes were made.
      - Commit the transaction.
      - Verify that changes persist after commit.